website/docs/Features/accessServer.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 1
```

Comment:
This affects the order of the sidebar entries and should not be duplicated.
accessContent.md is 1.
Would you please re-arrange the order as appropriate? You may have to change other files in this folder.

Reply:
Ah, good catch. I think I got the order right in https://github.com/sassoftware/vscode-sas-extension/pull/1203/commits/37db855d5dc4b685119dc99727695530febed8eb